packages/main/bundle.common.js

```diff
@@ -80,9 +80,13 @@ import TableCell from "./dist/TableCell.js";
 import TextArea from "./dist/TextArea.js";
 import TimeSelection from "./dist/TimeSelection.js";
 import TimePicker from "./dist/TimePicker.js";
+import TimePickerClock from "./dist/TimePickerClock.js";
```

Comment:
Aren't these modules imported by TimPicker, TimePickerBase classes. Do we need them here imported explicitly?

Reply:
I saw at the end that ui5-time-selection-clocks, ui5-time-picker-clock are used standalone for testing purposes, so TimeSelectionClocks  and TimePickerClock are ok to remain.

But check ToggleSpinButton and TimePickerInternals imports

Reply:
Done, removed.